stock_shipment_schedule_report/reports/shipment_schedule_report_compute.py

```diff
@@ -105,6 +105,8 @@ def _get_product_ids(self, category_id):
                                                       categs else categs
             categ_ids = [categ.id for categ in categs]
             domain.append(('categ_id', 'in', categ_ids))
+        if self.website_published:
+            domain.append(('website_published', '=', True))
```

Comment:
Please also add `('company_id', '=', self.env.user.company_id.id),` to the initial definition of the domain for products to make it:
```
        domain = [
            ('company_id', '=', self.env.user.company_id.id),
            ('sale_ok', '=', True),
            ('type', '=', 'product'),
            ('active', '=', True)
        ]
```